lib/nerves_livebook/dependencies.ex

```diff
@@ -0,0 +1,14 @@
+defmodule NervesLivebook.Dependencies do
+  def dependency_entries() do
```

Comment:
```suggestion
  def packages() do
```

We changed naming from `dependency_entries` to `packages` and apparently I only updated the config snippet in the original issue :)

We should be able to generate all of these entries on the fly, right?

Reply:
Haha, cool!

> I think that I'll need to investigate later how best to do this.

We should be able to generate the info based on `Application.spec`, but we need the list of relevant dependencies from `mix.exs` somehow.